org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/ILanguageServerAccess.java

```diff
@@ -33,15 +33,15 @@
  */
 public interface ILanguageServerAccess {
 	class Context {
-		private final Resource resource;
+		private final XtextResource resource;
```

Comment:
Why did you change this?

Reply:
because org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/DiagnosticResolution.java had a variable declared with XtextResource, I have downgraded it to a Resource in the last version so that this is not needed.